mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/CompletionRecord.java

```diff
@@ -25,10 +25,13 @@
     private int score;
 
     @Field
+    private int completionCount;
+
+    @Field
     private boolean sentNotification;
 
     @Field
-    private int completionCount;
+    private String lastDeliveryStatus;
```

Comment:
Mostly because we don't have any business logic to do based on this field. This just serves as a meta store for reporting (if there is a use case for it) and I optimized it for not having another nested entity table...maybe prematurely? :eyes: